nifi-nar-bundles/nifi-azure-bundle/pom.xml

```diff
@@ -26,12 +26,9 @@
     <packaging>pom</packaging>
 
     <properties>
-        <azure-storage.version>8.6.6</azure-storage.version>
-        <azure.core.version>1.26.0</azure.core.version>
-        <azure.identity.version>1.4.5</azure.identity.version>
-        <!-- azure-identity depends on msal4j transitively, keep these versions consistent -->
-        <msal4j.version>1.11.0</msal4j.version>
-        <azure-cosmos.version>4.26.0</azure-cosmos.version>
+        <azure.sdk.bom.version>1.2.4</azure.sdk.bom.version>
```

Comment:
Could you please bump the bom version to the latest `1.2.6`?
I still have troubles with restarting the Consume processor (no messages processed after restart) but it seems to work with that version.